### PR TITLE
feat(inlince preboot code) change press enter and click on buttons ev…

### PR DESCRIPTION
**Pull request by thebarada**
…ents to submit form

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/preboot/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What modules are related to this pull-request**
- [ ] server side
- [ ] client side
- [x] inline
- [ ] build process
- [ ] docs
- [ ] tests

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Change event selectors

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Comment by thebarada** [comment]
FYI
in my project, when I pressed enter form submitted though

File: src/server/inline.preboot.code.ts [hunk 1]
```diff
@@ -19,12 +19,11 @@ export const defaultOptions = <PrebootRecordOptions>{
     },
     {selector: 'select,option', events: ['change']},
 
-    // when user hits return button in an input box
+    // when user submit form (press enter, click on button/input[type="submit"])
     {
-      selector: 'input',
```
Comment (jeffwhelpley) [inline src/server/inline.preboot.code.ts]: It looks like you removed the default for when someone hits the return key in an input box. I agree with you that when someone has an input box within a form tag, what you have will work. However, not everyone actually uses a form tag these days and they still need the case where you hit the return key from within an input box that is not part of an HTML form.

Would it work for you to keep the old listener and add a new one for form submit?

Comment (thebarada) [inline src/server/inline.preboot.code.ts]: ok, I'll keep old listener, thank you for observation

**Comment by thebarada** [comment]
I signed it!

File: src/server/inline.preboot.code.ts [hunk 2]
```diff
@@ -28,19 +28,19 @@ export const defaultOptions = <PrebootRecordOptions>{
...
+
     // for tracking focus (no need to replay)
     {
       selector: 'input,textarea',
       events: ['focusin', 'focusout', 'mousedown', 'mouseup'],
       noReplay: true
-    },
-
-    // user clicks on a button
-    {
```
Comment (jeffwhelpley) [inline src/server/inline.preboot.code.ts]: Sorry, one more change. for the same reason as the other change we still need in an event listener for buttons. I think your form submission event listener will cover the `input[type="submit"]` but not a button which has no form. So, just add back in:

```
{
  selector: 'button',
  events: ['click'],
  preventDefault: true,
  freeze: true
}
```

Comment (thebarada) [inline src/server/inline.preboot.code.ts]: done

**Pull request merged**